tmt/plans/integration.fmf

```diff
@@ -3,15 +3,31 @@ provision:
   image: $@{test_disk_image}
 prepare:
   # Replace package mode with image mode
+  # DO NOT RUN ON Github or Locally
+  # Run on Packit and Gating only
+  - how: install
+    package:
+      - podman
+      - skopeo
+      - jq
+      - bootc
+      - system-reinstall-bootc
+      - expect
+      - ansible-core
+      - zstd
+    when: running_env != unify
   - how: shell
     script:
-      - pwd && ls -al
-      - if [[ -d hack ]]; then cd hack && ./provision-packit.sh; fi
+      - mkdir -p bootc && cp /var/share/test-artifacts/*.src.rpm bootc
+      - cd bootc && rpm2cpio *.src.rpm | cpio -idmv && rm -f *-vendor.tar.zstd && zstd -d *.tar.zstd && tar -xvf *.tar -C . --strip-components=1 && ls -al
```

Comment:
Ah. But if we do this, do we need the `dnf builddep` per above at all? I don't think so...